script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -316,118 +346,141 @@ And there's over fifty of these monsters...?
 [sha:f6944476c84780f89d1f7c544dbe84273bba2db1]{
 -- Page 599, Offset 18076.
 -- 「ほら、やっぱり。
-"I knew it.
+"Aha, I knew it.
 }
 [sha:e3daea629ec69f24f54e0a0cff983d20b008909f]{
 -- Page 599, Offset 18077.
 -- 殺したくて血が騒いできた、殺人鬼さん？」
-You're getting excited about wanting to kill them, aren't you, Mr. Murderer?"
+// Keeping the sanguine reference
+Did the desire to kill whip your blood into a frenzy, Mr. Murderer? "
 }
 [sha:bf925940fee76dfbac28d879dd75697d5d48ae9f]{
 -- Page 599, Offset 18078.
 -- 「騒いでない。こういうのは怖じ気づいてるって言うんだぜ、吸血鬼さん」
-"I'm not excited. This is what we refer to as fear, Ms. Vampire."
+"Far from it. This is what we refer to as being petrified with fear, Ms. Vampire."
 }
 [sha:fb5defe4e1cb192bc34fafb119827210c9a035fc]{
 -- Page 600, Offset 18079.
 -- 人間の尺度で返答する。
+// TODO(ross): better fit?
 I respond from a human's standpoint.
 }
 [sha:ae8707c32ad9f86b7afdd67d1020c676464b7b94]{
 -- Page 600, Offset 18080.
 -- 協力するとは決めたものの、このあたりきちんと主張しておきたい。人間にできない事はできないのである。
-Although I decided that I would cooperate, I want to properly emphasize this fact. The fact that humans cannot do the impossible.
+I may have decided to help her out, but I want to make myself absolutely clear on this.
+ There are some things that humans simply cannot do.
 }
 [sha:cac1f3e52ee9c8b06161bb5f5e7e0a420570b1f6]{
 -- Page 601, Offset 18081.
 -- 「それで、いよいよ本題のⅨ階梯だけど。
-"And so, we're finally at the main topic that is Rank IX.
+"So we've finally reached the topic at hand, Rank IX.
 }
 [sha:cfcf9ec1b8832ee4392b051ee3871a0ddea31d7d]{
 -- Page 601, Offset 18082.
 -- この二十七祖ってヤツらが、死徒の頂点でいいんだよな？」
-We can say that these Twenty-Seven Ancestors are at the top of the Dead Apostles, right?"
+// Not sure I've seen ~でいい for clarification before? I guess it makes sense?
+These Twenty-Seven Ancestors represent the apex of the Dead Apostles, right?"
 }
 [sha:9a5a6992d57cb904fbf0bdd7018afc5d9a4007dd]{
 -- Page 602, Offset 18083.
 -- 「そうよ。非常食の立場から抜け出し、まったく違うモノとして独立を勝ち取った古い死徒たち。
-"Correct. They are ancient Dead Apostles that broke out from the position of eating for survival and won their independence as completely different beings.
+// 非常食の立場 doesn't appear to be an idiom, so inclined to translate it literally
```

Comment:
I think it makes sense literally and it's definitely not an idiom (no permutations bring up real results on google)